eftcamb/04p4_exponential_parametrizations_1D.f90

```diff
@@ -0,0 +1,224 @@
+        type(EFTCAMB_timestep_cache), intent(in), optional :: eft_cache !< the optional input EFTCAMB cache
+        real(dl) :: ExponentialParametrized1DSecondDerivative           !< the output value
+
+        ExponentialParametrized1DSecondDerivative = self%exponential_value_1*self%exponential_value_2*(self%exponential_value_2-1._dl)*x**(self%exponential_value_2-2._dl)*Exp(self%exponential_value_1*x**self%exponential_value_2) &
+                 & +self%exponential_value_1**2*self%exponential_value_2**2*x**(2._dl*self%exponential_value_2 -2._dl)*Exp(self%exponential_value_1*x**self%exponential_value_2)
+
+    end function ExponentialParametrized1DSecondDerivative
+
+    ! ---------------------------------------------------------------------------------------------
+    !> Function that returns the third derivative of the function.
+    function ExponentialParametrized1DThirdDerivative( self, x, eft_cache )
+
+        implicit none
+
+        class(exponential_parametrization_1D)              :: self      !< the base class
+        real(dl), intent(in)                               :: x         !< the input scale factor
+        type(EFTCAMB_timestep_cache), intent(in), optional :: eft_cache !< the optional input EFTCAMB cache
+        real(dl) :: ExponentialParametrized1DThirdDerivative            !< the output value
+
+        ExponentialParametrized1DThirdDerivative = self%exponential_value_1*self%exponential_value_2*(self%exponential_value_2-1._dl)*(self%exponential_value_2-2._dl)*x**(self%exponential_value_2-3._dl)*Exp(self%exponential_value_1*x**self%exponential_value_2) &
+                 & +self%exponential_value_1**2*self%exponential_value_2**2*(self%exponential_value_2 -1._dl)*x**(2._dl*self%exponential_value_2 -3._dl)*Exp(self%exponential_value_1*x**self%exponential_value_2) &
+                 & +self%exponential_value_1**2*self%exponential_value_2**2*x**(3._dl*self%exponential_value_2 -3._dl)*Exp(self%exponential_value_1*x**self%exponential_value_2) &
+                 & +self%exponential_value_1**2*self%exponential_value_2**2*2._dl*(self%exponential_value_2 -1._dl)*x**(2._dl*self%exponential_value_2 -3._dl)*Exp(self%exponential_value_1*x**self%exponential_value_2)
+
+    end function ExponentialParametrized1DThirdDerivative
+
+    ! ---------------------------------------------------------------------------------------------
+    !> Function that returns the integral of the function
+    function ExponentialParametrized1DIntegral( self, x, eft_cache )
+
+        implicit none
+
+        class(exponential_parametrization_1D)              :: self      !< the base class
+        real(dl), intent(in)                               :: x         !< the input scale factor
+        type(EFTCAMB_timestep_cache), intent(in), optional :: eft_cache !< the optional input EFTCAMB cache
+        real(dl) :: ExponentialParametrized1DIntegral                   !< the output value
+
+        !< No analytic solution >!
```

Comment:
Mathematica does not find it? If this is the case we need to implement a numerical integral...sigh...